OptimizelySDKCore/OptimizelySDKCore/OPTLYProjectConfig.h

```diff
@@ -165,7 +165,7 @@ __attribute((deprecated("Use OPTLYProjectConfig initWithBuilder method instead."
  */
 - (BOOL)setForcedVariation:(nonnull NSString *)experimentKey
                     userId:(nonnull NSString *)userId
-              variationKey:(nonnull NSString *)variationKey;
+              variationKey:(nullable NSString *)variationKey;
```

Comment:
@thomaszurkan-optimizely @mikeng13 This is an issue in the existing SDK, while we send nil, application crashed. Fixing in this PR.

Reply:
So, the reason this should allow nil is that it is a way of removing a forced variation.  The forced variation is a in memory map. Good job on catching that and fixing it!